lua/astrocommunity/pack/clojure/clojure.lua

```diff
@@ -0,0 +1,54 @@
+-- Clojure support with Conjure plugin
+local utils = require "astronvim.utils"
+return {
+  -- Clojure Language Server
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clojure_lsp") end,
+  },
+  -- Clojure parser
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clojure")
+      end
+    end,
+  },
+  -- Parinfer parens management for Clojure
+  {
+    "gpanders/nvim-parinfer",
+    ft = { "clojure" },
+    config = function()
```

Comment:
```suggestion
    init = function()
```

This should be an `init` function I'm pretty sure